src/MockTableGenerators.jl

```diff
@@ -1,6 +1,7 @@
 module MockTableGenerators
 
 using Dates: Period
+using Random
```

Comment:
I like to be explicit about imports to know when they can be removed
```suggestion
using Random: AbstractRNG
```